index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html style=" background-color:aqua;
+text-align:center;
+" >
+    <head>
+    
+    <link rel="stylesheet" href="project-1.css">
+    
+    </head>
+    <body>
+    <div class="mid" >
```

Comment:
Use the appropriate tags for each element (e.g., links, titles, etc.) and use HTML5 semantic tags (e.g., header, nav) over divs. For proper understanding.